bertopic/_save_utils.py

```diff
@@ -363,6 +371,13 @@ def save_config(model, path: str, embedding_model):
 
     return config
 
+def check_has_visual_aspect(model):
```

Comment:
This is heavily duplicating code in `save`, but I didn't want to take the liberty of refactoring this out!

Reply:
Feel free to refactor stuff but this is fine by me for now!